docs/table/mixin_columns.rst

```diff
@@ -141,6 +141,16 @@ You can conveniently convert |Table| to |QTable| and vice-versa::
    ordinary `~astropy.table.Table` then it gets converted to an ordinary
    `~astropy.table.Column` with the corresponding ``unit`` attribute.
 
+.. attention::
+
+  When a column of ``int`` ``dtype`` is converted to `~astropy.units.Quantity`,
+  its ``dtype`` is converted to ``float``.
+
+  For example, for a quality flag column of ``int``, if it is
+  assigned with the unit `~astropy.units.dimensionless_unscaled`, it will still
```

Comment:
RTD failed because of invalid linking here.

```
docs/table/mixin_columns.rst:149: WARNING:
py:obj reference target not found: astropy.units.dimensionless_unscaled
```

Reply:
hmm. running a test

Reply:
... I'm not sure why `~astropy.units.dimensionless_unscaled` isn't working. I think it might be the way it's generated on import. Worth further investigation.

In the meantime, @orionlee has a good solution to link to the docs.

Reply:
thanks - it's now linked to dimensionless unit's doc.